### PR TITLE
Provision all meta environments

### PR DESCRIPTION
**Pull request by adamsilverstein**
See https://github.com/WordPress/meta-environment/blob/master/docs/install.md

Testing this now building images for WCUS

### CODE REVIEW

File: resources/vvv-custom.yml [hunk 1]
```diff
@@ -25,8 +25,18 @@ sites:
   # The following commented out site configuration will create a environment useful
   # for contributions to the WordPress meta team:
   
-  wordpress-meta-environment:
-    repo: https://github.com/WordPress/meta-environment.git
+ wordpress-meta-environment:
```
Comment (cfoellmann) [inline resources/vvv-custom.yml]: did you keep the space counts? alignment WILL be a problem here

Comment (adamsilverstein) [reply resources/vvv-custom.yml]: oops, lemme redo that, copy pasta error from the readme

**Comment by adamsilverstein** [comment]
does that look right?

**Comment by cfoellmann** [comment]
that looks fine to me

**Comment by adamsilverstein** [comment]
testing it out, I'll let you know when it completes. Thanks for the excellent tool, much better than doing it by hand!

**Comment by adamsilverstein** [comment]
The build worked fine after i downgraded to vagrant 1.8.5 locally. When trying to build, I get an error installing the vagrant-triggers plugin with vagrant 2.x - it works fine with vagrant 1.8.5 - 

https://cl.ly/2J1T3U0p0X1M

possibly related: https://github.com/hashicorp/vagrant/issues/8210

**Comment by tomjn** [comment]
@adamsilverstein you have to do a full uninstall vagrant when upgrading to vagrant 2.0 using the uninstall tool in the vagrant 2 dmg before upgrading to 2.0

**Review by tomjn — APPROVED** [review]

**Comment by adamsilverstein** [comment]
Ok I’ll give that a try!

**Comment by cfoellmann** [comment]
@tomjn do you know if there is some uninstaller for Windows, too? I broke my vagrant by updating from 1.9.8 to 2.0.1 and couldn’t recover until downgrading to 1.9.5

**Comment by tomjn** [comment]
@cfoellmann I don't know, there should be ( Add/Remove Programs? ), but I'm not really a Windows user these days

**Pull request merged**